solarforecastarbiter/reports/figures/plotly_figures.py

```diff
@@ -373,11 +392,18 @@ def _plot_obs_timeseries(fig, timeseries_value_df, timeseries_meta_df):
         )
         if data['observation_values'].isnull().all():
             continue
+        # Append the interval length and labelling to each observation
+        # to ensure unique names
+        interval_text = formatted_interval(metadata["interval_length"])
+        label_text = metadata['interval_label']
+        observation_legend_name = _legend_text(
+           f'{ metadata["observation_name"]} {interval_text} {label_text}'
```

Comment:
Do we really want the space at the start of the string? Add comment in code if so.

Reply:
The space is actually just in the fstring and not in the output, but I'll clean this up.